datalad/distribution/install.py

```diff
@@ -285,8 +282,8 @@ class Install(Interface):
     @staticmethod
     @datasetmethod(name='install')
     def __call__(
+            source,
             path=None,
-            source=None,
```

Comment:
how would it translate to cmdline API?... my initial suspicion seems to be correct that "unfavorably":
```
$> cd ~/datalad/openfmri 

$> datalad install ds*
2016-09-29 20:41:01,556 [ERROR  ] unknown arguments: ['ds000003', 'ds000005', 'ds000006', 'ds000007', 'ds000008', 'ds000009', 'ds000011', 'ds000017A', 'ds000017B', 'ds000030', 'ds000031', 'ds000051', 'ds000052', 'ds000053', 'ds000101', 'ds000102', 'ds000105', 'ds000107', 'ds000108', 'ds000109', 'ds000110', 'ds000113', 'ds000113b', 'ds000113c', 'ds000113d', 'ds000114', 'ds000115', 'ds000116', 'ds000117', 'ds000119', 'ds000120', 'ds000121', 'ds000122', 'ds000133', 'ds000140', 'ds000148', 'ds000149', 'ds000157', 'ds000158', 'ds000168', 'ds000170', 'ds000171', 'ds000172', 'ds000174', 'ds000177', 'ds000200', 'ds000201', 'ds000202', 'ds000203', 'ds000204', 'ds000205', 'ds000206', 'ds000208'] (main.py:236)
usage: datalad install [--version] [-h] [-l LEVEL] [-p {condor}] [-d DATASET]
                       [-g] [-D DESCRIPTION] [-r] [--recursion-limit LEVELS]
                       [--if-dirty {fail,save-before,ignore}] [--nosave]
                       [--reckless] [--git-opts STRING]
                       [--git-clone-opts STRING] [--annex-opts STRING]
                       [--annex-init-opts STRING]
                       SOURCE [PATH]
```
so breaks current behavior (to begin with).